Trustroots/trustroots
# Migrate "Report Member" link to React

#### Proposed Changes

* Migrate `./modules/support/client/directives/tr-report-member.client.directive.js` to React.
* We keep the AngularJS directive, because it's used in multiple not-yet-migrated places

#### Testing Instructions

* Open the user profile (other than self) and see:
  - there is still a Report Member link under the profile basic view
  - there is still a Report Member link in `/messages/:username`
  - the link redirects to support form with reported user's name filled
  - the title shows as expected
  - see that i18n works (but we don't have translations to test this)

Continuous work on #1019
